cpp/include/cudf/io/parquet_schema.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018-2025, NVIDIA CORPORATION.
+ * Copyright (c) 2025, NVIDIA CORPORATION.
```

Comment:
This seems incorrect. 2018-2025 is what it should be, right?

Reply:
We have recently (in 2025) created this new public header file and moved footer structs from `src/` here.